addons/mail/static/src/components/call_main_view/call_main_view.js

```diff
@@ -17,6 +17,18 @@ export class CallMainView extends Component {
         useComponentToModel({ fieldName: 'component' });
         useRefToModel({ fieldName: 'tileContainerRef', refName: 'tileContainer', });
         useUpdateToModel({ methodName: 'onComponentUpdate' });
+
+        onMounted(() => {
+            this.resizeObserver = new ResizeObserver(() => {
+                this.callMainView.onComponentUpdate();
```

Comment:
This should call "onResize" kind of method, not the update one. If the code is common to both it should be split on a separate function.

Reply:
I think that `onComponentUpdate` only existed for this flow, so I suppose that I can just rename it, 

@alexkuhn you made the change that added `onComponentUpdate`,  does it makes sense for you?

Reply:
ah nevermind, it's also used for when new tiles are added which doesn't involve a resize